### ISSUE TITLE
Int64 serialization culture problem

### ISSUE THREAD

**Issue by keesschollaart81** [Azure/azure-documentdb-net#42]
When using the SDK to query (with Linq) for a document I ran into some problems with filtering a query on a Int64 property.

I figured out that in the REST request the Int64 is serialized as (for example) 6,35430269777489E+17 which results in a server error. It seems that this is because of my local machines culture settings (nl-NL).

I've created an example application to show the bug, or am I missing something?
https://github.com/keesschollaart81/DocumentDbSerializationTest

**Comment by ryancrawcour** [comment]
If you do the same query doing SQL, instead of LINQ does it work as expected?

**Comment by keesschollaart81** [comment]
I've updated my test application to answer your question. Running the code results in:
```
Via Linq lambda (en-GB)
  {"query":"SELECT * FROM root WHERE (root.created > 6.35431933725259E+17) "}
  Successful!

Via Linq lambda (nl-NL)
  {"query":"SELECT * FROM root WHERE (root.created > 6,35431933725259E+17) "}
  Failed: Message: {"errors":[{"severity":"Error","location":{"start":42,"end":43},"code":"SC1001","message":"Syntax error, incorrect syntax near ','."}]} ...

Via SqlQuerySpec query (en-GB)
  {"query":"SELECT * FROM root WHERE (root.created > @lastYear)","parameters":[{"name":"@lastYear","value":635431933725259404}]}
  Successful!

Via SqlQuerySpec query (nl-NL)
  {"query":"SELECT * FROM root WHERE (root.created > @lastYear)","parameters":[{"name":"@lastYear","value":635431933725259404}]}
  Successful!

Via custom generated query (en-GB)
  {"query":"SELECT * FROM root WHERE (root.created > 6.3543193373E+017)"}
  Successful!

Via custom generated query (nl-NL)
  {"query":"SELECT * FROM root WHERE (root.created > 6,3543193373E+017)"}
  Failed: Message: {"errors":[{"severity":"Error","location":{"start":42,"end":43},"code":"SC1001","message":"Syntax error, incorrect syntax near ','."}]} ...
```

In the last tests, I reproduced the problem by creating my own query and formatting the Int64 with the ```E``` formatter ('scientific'). 

Apparently the SqlQuerySpec parameters get serialized by another serializer?

**Comment by ryancrawcour** [comment]
Thanks. Will take a look shortly.

**Comment by ryancrawcour** [comment]
This has been confirmed as an issue in the SDK with the linq provider. We will address this in a future release.

**Comment by ryancrawcour** [comment]
This has been fixed with 1.4.1 of the NuGet package. http://www.nuget.org/packages/Microsoft.Azure.DocumentDB/1.4.1
